test/certs.js

```diff
@@ -32,28 +31,10 @@ describe('Connection to server with empty subject', function () {
     }
   });
 
-  // This test is mainly to keep track of the original Node.Js issue. When this test fails you can safely remove ../lib/tls.js
```

Comment:
> When this test fails you can safely remove ../lib/tls.js
could you please remove all references to `./lib/tls.js` and SSL_ENABLE_EMPTY_SUBJECT?

Reply:
done.